### PR TITLE
mini CLR rounds on newsfeed - READY FOR REVIEW

### PR DESCRIPTION
**Pull request by owocki**
##### Description

This is what this looks like:
Screen Shot 2020-02-11 at 7 18 43 PM

The bones are done done:
- [X] mechanisms to create /schedule match rounds
- [X] mechanisms to continually refresh match rankings
- [X] mechanism to automatically make a CLR payout
- [X] mechanism to follow users straight on newsfeed

Still gotta put the finishing touches on here:
- [x] show CLR estimates on newsfeed items (not just leaderboard)
- [x] make estimates refresh when new tip is made (either ajax or fresh)
- [x] make tips non alpha 
- [ ] a TLDR how it works module (need help from design)
- [ ] what do we do with the follow data? do we highlight it into the various tab types?
- [x] follow buttons everywhere on newsfeed  
- [x] create Earnings, newsfeed items, email when payout happys.  celebrate it!

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/5941

##### Testing

tested locally  

should be merged along with https://github.com/gitcoinco/web/pull/5916/

### CODE REVIEW

**Comment by owocki** [comment]
build is failing bc of some rinkeby issue

```
   
        if http_error_msg:
>           raise HTTPError(http_error_msg, response=self)
E           requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://rinkeby.infura.io/
```

**Comment by danlipert** [comment]
> build is failing bc of some rinkeby issue
> 
> ```
>    
>         if http_error_msg:
> >           raise HTTPError(http_error_msg, response=self)
> E           requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://rinkeby.infura.io/
> ```

@owocki You need to use a real infura account now with the app id and such https://community.infura.io/t/legacy-key-deprecation/443

File: app/app/settings.py [hunk 1]
```diff
@@ -755,3 +755,6 @@
 RE_MARKET_LIMIT = env.int('RE_MARKET_LIMIT', default=2)
 MINUTES_BETWEEN_RE_MARKETING = env.int('MINUTES_BETWEEN_RE_MARKETING', default=60)
 
+MINICLR_ADDRESS = env('MINICLR_ADDRESS', default='0x00De4B13153673BCAE2616b67bf822500d325Fc3')
+MINICLR_PRIVATE_KEY = env('MINICLR_PRIVATE_KEY', default='0x00De4B13153673BCAE2616b67bf822500d325Fc3')
```
Comment (danlipert) [inline app/app/settings.py]: Probably worth generating a throwaway keypair here for defaults

**Comment by owocki** [comment]
> @owocki You need to use a real infura account now with the app id and such https://community.infura.io/t/legacy-key-deprecation/443

sure; is there a .TRAVIS_ENV somewhere we can update?

**Comment by owocki** [comment]
61987bc2721990a11e6ca6f33d265da7594f264c handles rinkeby key

**Pull request merged**